.github/workflows/dash.yaml

```
@@ -25,6 +25,8 @@ on:
   pull_request:
     branches:
       - main
+    paths:
+      - "package-lock.json"
```

we should include the package.json as well
```suggestion
      - package-lock.json
      - package.json
```

---

thx for the hint